packages/-build-infra/src/addon-build-config-for-data-package.js

```diff
@@ -111,6 +110,9 @@ function addonBuildConfigForDataPackage(PackageName) {
     },
 
     treeForAddon(tree) {
+      if (typeof this._additionalAddonFiles === 'function') {
```

Comment:
If this is only needed by `ember-data` package, can we put a custom `treeForAddon` there instead and merge in the file after calling the original implementation here?

Reply:
since we don't have class based inheritance, no :(

Reply:
We can definitely do it

```js
module.exports = Object.assign(addonBaseConfig, {
 // other stuff here

  treeForAddon() {
    let tree = addonBaseConfig.treeForAddon.apply(this, arguments);
    return merge([tree,version()]);
  }
});
```

Reply:
👍